### PR TITLE
Open links in new tab

### PR DESCRIPTION
**Pull request by aravindet**
- Switch all buttons to links with href coming from a computed property (and _blank for desktop links)
- Add computed properties mailtourl, gmailurl, yahoourl and fullmailtourl (for mobile)
- Clean up URL generation code

Behavior Changes

This affects desktop users with old browsers.

If copy fails on desktop (i.e. the browser does not support execCommand copy), it will display the message asking the user to copy manually and click the Gmail/Yahoo/Other button again. The second time the button is clicked, it opens a tab even if copying failed. This is different from current behavior where the page navigates to the previously chosen email client on copy.

Caveats
- Firefox does not open mailto: links in new tab

### CODE REVIEW

File: _includes/campaign-form.html [hunk 1]
```diff
@@ -54,17 +54,21 @@
...
-      v-on:click.stop.prevent="sendEmail('yahoo')">Yahoo!</button>
-    <button class="btn-email button-unstyled" v-if="!mobile"
-      v-on:click.stop.prevent="sendEmail('other')">Other</button>
+    <a class="btn btn-email btn-primary" v-if="!mobile"
+      v-bind:href="gmailurl" target="_blank"
+      v-on:click.stop="sendGmail">GMail</a>
+    <a class="btn btn-email btn-primary" v-if="!mobile"
+      v-bind:href="yahoourl" target="_blank"
+      v-on:click.stop="sendYahoo">Yahoo!</a>
+    <a class="btn btn-email btn-primary" v-if="!mobile"
```
Comment (aravindet) [inline _includes/campaign-form.html]: Without btn-primary, the link does not look like a button.

File: _includes/campaign-form.html [hunk 2]
```diff
@@ -54,17 +54,21 @@
 
     <label v-if="campaign==='mp'" for="service-name">[{$t("contempt_notice_msg")}] : <span class="service-name">[{personName}], [{$t("constituencies." + constituencyCode)}]</span></label>
 
-    <button id="btn-email-mobile" class="btn-email btn-primary" v-if="mobile"
-      v-on:click.stop.prevent="sendEmail()">[{$t("send_email")}]</button>
+    <a id="btn-email-mobile" class="btn btn-email btn-primary" v-if="mobile"
+      v-bind:href="fullmailtourl"
```
Comment (aravindet) [inline _includes/campaign-form.html]: No _blank on mobile.

Comment (captn3m0) [reply _includes/campaign-form.html]: On mobiles we are using `mailto:` anyway, so it doesn't matter

Comment (aravindet) [reply _includes/campaign-form.html]: Yup, didn't want to open a new tab unnecessarily.

**Pull request merged**